README.md

```diff
@@ -66,7 +66,7 @@
 
 We are all about building bridges to empower others towards the direction of their lives. With **Legesher**, our bridges allow people to code in their native language. Legesher is designed for individuals where English is a second language and for diverse teams that span across many other barriers already.
 
-**Legesher** is a tool that enables anyone to code in their native language. In order to do that we have several projects within the **Legesher** theme to support such an mission. We built **Legesher** on top of [Atom](atom.io), a hackable text editor, and [Git w/ GitHub](https://github.com), a version control tool, in order to give users as much of a personal invitation to programming as we could.
+**Legesher** is a tool that enables anyone to code in their native language. In order to do that we have several projects within the **Legesher** theme to support such a mission. We built **Legesher** on top of [Atom](atom.io), a hackable text editor, and [Git w/ GitHub](https://github.com), a version control tool, in order to give users as much of a personal invitation to programming as we could.
```

Comment:
Thanks for finding this @Sayanta66. Would love to merge this, but it needs to have a little more significant changes for Hacktoberfest. Can I open an issue with some meaningful changes for you to complete?

Reply:
Yes please. I can do copy-editing quite well. You can create and assign one to me

Reply:
Awesome, I'll create one and assign it to you. If you'd like I can create multiple issues for you, as the README is the same for most of the repos in this project.

Reply:
Yeah please do....assign me after creation. I will complete the work. Thanks a lot for giving this opportunity.